graphql/admin/endpoints_ee.go

```diff
@@ -143,6 +143,19 @@ const adminTypes = `
 		Set to true to allow backing up to S3 or Minio bucket that requires no credentials.
 		"""
 		anonymous: Boolean
+
+		"""
+		oneNamespaceOnly should be set if only one namespace is to be restored from
+		the backup. The namespace value should be set into the fromNamespace parameter.
+		The given namespace is always restored into namespace 0 in the cluster.
+		"""
+		oneNamespaceOnly: Boolean
```

Comment:
Do we need `oneNamespaceOnly`? Can we use `fromNamespace > 0` to imply the same?